nasigoreng.html

```diff
@@ -1,94 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
 <head>
-
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nasi Goreng</title>
```

Comment:
add favicon under title
```suggestion
        <link rel="icon" type="image/x-icon" href="./img/logo_small.png">
```